frontend/www/js/omegaup/components/common/LogoutConfirmation.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+  <b-modal
+    v-model="showModal"
+    :title="T.logoutConfirmationTitle"
+    :ok-title="T.wordsYes"
+    :no-title="T.wordsNo"
+    ok-variant="primary"
+    no-variant="secondary"
```

Comment:
Same case as the previous comment:
```suggestion
    cancel-variant="secondary"
```